forte/mrdsrg-spin-integrated/dsrg_mrpt2.h

```diff
@@ -175,6 +178,294 @@ class DSRG_MRPT2 : public MASTER_DSRG {
+     */
+    void solve_z();
+    /**
+     * Initialize and solve the multiplier Sigma.
+     *
+     * Sigma: constraint of the one-body DSRG amplitude (T1) definition.
+     * Solved directly.
+     */
+    void set_sigma();
+    /**
+     * Initialize and solve the multiplier Xi.
+     *
+     * Xi: constraint of the renormalized Fock matrix (F1) definition.
+     * Solved directly.
+     */
+    void set_xi();
+    /**
+     * Initialize and solve the multiplier Tau.
+     *
+     * Tau: constraint of the two-body DSRG amplitude (T2) definition.
+     * Solved directly.
+     */
+    void set_tau();
+    /**
+     * Initialize and solve the multiplier Kappa.
+     *
+     * Kappa: constraint of the renormalized ERIs (\tilde{V}) definition.
+     * Solved directly.
+     */
+    void set_kappa();
+    /**
+     * Initialize and solve the multiplier Z (OPDM).
+     *
+     * Z: OPDM, constraint of the CASSCF reference.
+     * The core-core, virtual-virtual blocks and diagonal entries of the active-active
+     * blocks are solved directly based on other multipliers. The rest need be solved
+     * Through iterative approaches. Currently, we use LAPACK as the solver.
+     */
+    void set_z();
+    void set_alpha();
+    void set_CI();
+    /**
+     * Initializing the b of the Linear System Ax=b.
+     */
+    void set_b();
+    /**
+     * The core-core block of the OPDM Z.
+     */
+    void set_z_cc();
+    /**
+     * The virtual-virtual block of the OPDM Z.
+     */
+    void set_z_vv();
+    /**
+     * The diagonal entries of the active-active block of the OPDM Z.
+     */
+    void set_z_aa_diag();
+    /**
+     * Initialize and solve the multiplier W.
+     *
+     * W: EWDM, constraint of the orthonormal overlap integral.
+     * Solved directly after all other multipliers are solved.
+     */
+    void set_w();
+
+    /// Size of determinants
+    size_t ndets;
+    /// List of core MOs (Correlated)
+    std::vector<size_t> core_mos;
+    /// List of active MOs (Correlated)
+    std::vector<size_t> actv_mos;
+    /// List of virtual MOs (Correlated)
+    std::vector<size_t> virt_mos;
+    /// List of core MOs (Absolute)
+
+    std::vector<size_t> core_all;
+    /// List of active MOs (Absolute)
+    std::vector<size_t> actv_all;
+    std::vector<size_t> virt_all;
+
+    /// List of relative core MOs
+    std::vector<std::pair<unsigned long, unsigned long>,
+                std::allocator<std::pair<unsigned long, unsigned long>>>
+        core_mos_relative;
+    /// List of relative active MOs
+    std::vector<std::pair<unsigned long, unsigned long>,
+                std::allocator<std::pair<unsigned long, unsigned long>>>
+        actv_mos_relative;
+    /// List of relative virtual MOs
+    std::vector<std::pair<unsigned long, unsigned long>,
+                std::allocator<std::pair<unsigned long, unsigned long>>>
+        virt_mos_relative;
+
+    /// Dimension of different irreps
+    psi::Dimension irrep_vec;
```

Comment:
There are a _lot_ of class variables that still need to be renamed to end in an `_`. I know it's tedious, but it's an easy way to check if a variable is a class variable without checking the entire header file.